app/assets/stylesheets/modules/_events.css.scss

```diff
@@ -18,6 +18,16 @@
       color: $black;
     }
   }
+
+  .status-dropdown {
+    display: none;
+  }
+
+  #event_state {
+    width: 100%;
+    display: block;
+    margin-bottom:.25em;
+  }
 }
 
 
```

Comment:
@mbburch Let me know if you want help merging in master with this and forms.css.scss which have been renamed.